src/number_theory/function_field.lean

```diff
@@ -74,6 +74,26 @@ begin
     intros c x, convert (this (e.symm c) x).symm, simp only [e.apply_symm_apply] },
 end
 
+lemma polynomial.not_is_field : ¬ is_field Fq[X] :=
```

Comment:
```suggestion
lemma polynomial.not_is_field (R : Type*) [comm_ring R] [nontrivial R] : ¬ is_field R[X] :=
```
probably this should move to another file then too

Reply:
Thanks! I've moved this to `data/polynomial/div`.